apps/pie-docs/src/patterns/haptic-feedback/guidance/guidance.md

```diff
@@ -0,0 +1,139 @@
+---
+eleventyNavigation:
+    key: Guidance
+    parent: 'Haptic Feedback'
+    order: 2
+shouldShowContents: true
+---
+
+## About our haptic feedback types
+
+Our entire range of haptic feedback options relies on the system-default haptics provided by Apple in accordance with their [Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines).
+
+{% notification {
+  type: "warning",
+  message: "Please note that our default haptic feedback types **cannot be overridden or altered**."
+} %}
+
+---
+
+## Soft haptic feedback
+
+Soft haptic feedback can serve various purposes in order to enhance the overall user experience in a subtle and nuanced manner.
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/haptic-feedback/soft-haptic-feedback.svg",
+  alt: "Visual representation of soft haptic feedback.",
```

Comment:
I asked chatgtp for ideas for this alt text 😄 not sure if it conveys more meaning, so take this as a suggestion and if sounds weird we can leave it as it is. Adding the text for all haptic images:
"Vertical short orange bar intersected by a horizontal line, representing a single haptic feedback signal."
"Vertical long orange bar intersected by a horizontal line, indicating single haptic feedback signal."
"Two vertical orange bars, one short the next long, placed side by side and intersected by a horizontal line, symbolizing multiple haptic feedback signals."
"Two vertical orange bars, one longe the next short, placed side by side and intersected by a horizontal line, symbolizing multiple haptic feedback signals."
"Four vertical orange bars, the third one longer than the others, placed side by side and intersected by a horizontal line, symbolizing stronger haptic feedback signals."

Reply:
I think it's good idea to add more meaning to alt texts, I would only add some changes. If you are happy with these changes, I can add them to the code @maledr5 
- "Vertical short orange bar intersected by a horizontal line, representing `soft` haptic feedback signal."
- "Vertical long orange bar intersected by a horizontal line, indicating `medium` haptic feedback signal."
- "Two vertical orange bars, one short the next long, placed side by side and intersected by a horizontal line, representing `success` haptic feedback signals."
- "Two vertical orange bars, one longer, the next short, placed side-by-side and intersected by a horizontal line, representing `warning` haptic feedback signals."
- "Four vertical orange bars, the third one longer than the others, placed side by side and intersected by a horizontal line, symbolizing `error` haptic feedback signals."